README.md

```diff
@@ -136,6 +151,10 @@ onSortingChange (sortOrder) {…}
 
 A scenario where the loading state in the action bar might be useful is if there is some amount of processing that needs to be done in order to determine the state of the controls within the action bar. For example, determining whether a button should be enabled/disabled or shown/hidden. Note that the loading animation is the ring type from [ember-frost-core](https://github.com/ciena-frost/ember-frost-core).
 
+#### More... Button
+
```

Comment:
i was thinking we should document which buttons will get wrapped up into the `More...`.  Doesn't the order in the `controls` array matter?  Does your code exempt the first 4 `frost-button`s it encounters, then take the remaining `frost-button`s and wrap these into the `More...` button?

If so, change documentation to state thus.


take any `frost-button`s and wrap them up into a `More...` button

Reply:
Good point @lirwin.  @gkchestertron please either make this update or create an issue to capture the need to update the documentation.

Reply:
Good point - issue created to update readme and inline documentation to clarify.